### PR TITLE
Test cases for create, retrieve and delete

### PR DESCRIPTION
**Pull request by alekhyaved**

### CODE REVIEW

File: src/test/java/com/twitter/client/TwitterClientApplicationTests.java [hunk 1]
```diff
@@ -2,15 +2,59 @@
...
 	public void contextLoads() {
 	}
-
+	
+	@Autowired
+	TweetService tweetservice;
+	
+	@Test
+	public void createTweet() {
+		try {
```
Comment (chihhoang) [inline src/test/java/com/twitter/client/TwitterClientApplicationTests.java]: Test cases shouldn't have try/catch

File: src/test/java/com/twitter/client/TwitterClientApplicationTests.java [hunk 2]
```diff
@@ -2,15 +2,59 @@
...
 @RunWith(SpringRunner.class)
 @SpringBootTest
 public class TwitterClientApplicationTests {
 
 	@Test
 	public void contextLoads() {
 	}
-
+	
+	@Autowired
```
Comment (chihhoang) [inline src/test/java/com/twitter/client/TwitterClientApplicationTests.java:23]: We shouldn't rely on Twitter service in test. You can use @MockBean on this.

**Review by chihhoang — COMMENTED** [review]
Please see comments

**Review by chihhoang — COMMENTED** [review]
BTW this is not unit test. `@RunWith(SpringRunner.class)` indicates integration tests

File: src/test/java/com/twitter/client/TwitterClientApplicationTests.java (hunk 1 above)
Comment (alekhyaved) [reply src/test/java/com/twitter/client/TwitterClientApplicationTests.java]: modified without try/catch

**Review by chihhoang — APPROVED** [review]

**Pull request merged**